run.sh

```diff
@@ -99,9 +107,6 @@ while true; do
 done
 
 workload_1_golang() {
-  local SALT="$(date +'%s%N')"
-  local BIGQUERY_TABLE="${WORKLOAD}"
-  local RESULTS_FILE="${WORKLOAD}_results_${API}_${SALT}.csv"
   golang_benchmark_cli -p "${PROJECT}" \
                        -bucket "${BUCKET_NAME}" \
                        -defaults \
```

Comment:
This does not need to be done now, but this should probably be a parameter. 

We also have a bunch of other parameters that we will eventually want to be able to pass through - any ideas/plans on how we may do that other than adding a bunch of cli options? We will probably have some parameters that are only used for 1 language, so I can see the number of options getting out of hand.

Reply:
The CLI commands are added to pass through to the command but also set up any additional work; I've been thinking about this as well because each revision of a language CLI may introduce something that we don't want to always submit PRs to the repo just to update; I'll add this as a follow-up bug to address.